library/ecdh.c

```diff
@@ -57,6 +64,10 @@ static int ecdh_gen_public_restartable( mbedtls_ecp_group *grp,
                     mbedtls_ecp_restart_ctx *rs_ctx )
 {
     int ret;
+    ECDH_VALIDATE_RET( grp != NULL );
```

Comment:
Not a blocker: this is an internal function, the null checks are redundant.

Reply:
Fixing.